### PR TITLE
README: Add contribution guidelines with DCO

### PR DESCRIPTION
**Pull request by SmithChart**
We want to help new contributors help find their way to the 'Pull Request' tab.

Also we want contributions to contain a DCO.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -531,3 +531,47 @@ console via the debug UART adapter / the USB console provided by barebox
 and execute `wd -x` on the shell. E.g.:
 
     barebox@Linux Automation Test Automation Controller (TAC):/ wd -x
+
+Contributing
+------------
+
+Thank you for thinking about contributing to this project!
+Changes should be submitted via a
+`Github pull request <https://github.com/linux-automation/meta-lxatac/pulls>`_.
```
Comment (hnez) [inline README.md]: This looks like ReStructuredText syntax but the README is in Markdown.

Rendered result:

dco

File: README.md [hunk 2]
```diff
@@ -531,3 +531,47 @@ console via the debug UART adapter / the USB console provided by barebox
...
+    
+    (d) I understand and agree that this project and the contribution
+    are public and that a record of the contribution (including all
+    personal information I submit with it, including my sign-off) is
+    maintained indefinitely and may be redistributed consistent with
+    this project or the open source license(s) involved.
+
+Then you just add a line (using ``git commit -s``) saying:
+
+Signed-off-by: Random J Developer <random@developer.example.org>
```
Comment (hnez) [inline README.md]: Maybe also indent this so that it is shown as monospace text?

**Review by hnez — CHANGES_REQUESTED** [review]
Apart from the formatting issues this looks fine to me.

File: README.md (hunk 2 above)
Comment (jluebbe) [reply README.md]: As this is not code but normal text, I don't see the need to use monospaced.

File: README.md [hunk 3]
```diff
@@ -531,3 +531,46 @@ console via the debug UART adapter / the USB console provided by barebox
 and execute `wd -x` on the shell. E.g.:
 
     barebox@Linux Automation Test Automation Controller (TAC):/ wd -x
+
+Contributing
+------------
+
+Thank you for thinking about contributing to this project!
+Changes should be submitted via a
+[Github pull request](https://github.com/linux-automation/meta-lxatac/pulls>).
```
Comment (hnez) [inline README.md]: There is a stray trailing `>` here that breaks the link.

**Comment by hnez** [comment]
Could you also add a Contributing link to the table of contents at the beginning of the README?

**Comment by SmithChart** [comment]
> Could you also add a Contributing link to the table of contents at the beginning of the README?

Done.

**Review by hnez — APPROVED** [review]

**Comment by hnez** [comment]
Cool, looks good now. Thanks!

**Pull request merged**